source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/GenericImageLoaderAsync.java

```diff
@@ -126,6 +131,14 @@
 
         // Get image identifier
         Resources resources = context.getResources();
+
+        // For host app only provides image file name as url, host app can pass "package:[IMAGE NAME]" as replacement for
+        // meeting the valid URL format checking. Here we will remove this prefix to get file name.
+        if (url.startsWith("package:"))
+        {
+            url = url.replace("package:", "");
+        }
```

Comment:
As far as I know, the assets included with the project are always located in the drawable folder, so we can avoid the use of the baseImageUrl property from the hostConfig json

I think it's better to change this line (line 76)
if (m_imageBaseUrl == null || m_imageBaseUrl.isEmpty())
adding a check for the path not starting with package
if ((m_imageBaseUrl == null || m_imageBaseUrl.isEmpty()) && !path.startsWith("package:"))

And making a change here to assign imageBaseUrl with value "drawable/"

Reply:
Since the time you made this PR we figured out that we were doing something wrong about how the resource resolvers worked on android, we changed that to mimic the behavior of resource resolvers to the one in UWP and iOS. I say we should keep the "content:" option as is and just remove the "package:" option, as you can see in the mobile test app, that change is already there as a resource resolver